src/demangle.h

```diff
@@ -6,15 +6,19 @@
 #ifndef _DEMANGLE_H
 #define _DEMANGLE_H
 
+#include "rust-demangle.h"
```

Comment:
How exactly Rust symbols are demangled is an implementation detail; I don't think Rust-specific headers should appear in the declaration part of Demangle class. In fact, nothing is really needed from `rust-demangle.h` here - you can just put a forward declaration of `struct demangle`.